scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -83,11 +84,11 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
 
   def scalaJSToolsClasspath = T { scalaJSWorkerClasspath() ++ scalaJSLinkerClasspath() }
 
-  def fastLinkJS: Target[Report] = T {
+  def fastLinkJS: Target[Report] = T.persistent {
```

Comment:
Asked about this on the Scala.js Discord:
<img width="771" alt="Schermata 2022-05-05 alle 13 21 08" src="https://user-images.githubusercontent.com/5793054/166913286-fcb51b3d-3150-4c44-97d5-148fb2d10066.png">

Reply:
(Hah, a nice example why discord isn't the best option for an open community. You can't easily link.)

Could you add a small comment about that behavior. This makes reasoning about the `T.persistent` much easier.